hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+:func:`~hypothesis.strategies.from_type` now correctly handles :pypi:`annotated-types`
+annotations on :class:`typing.TypedDict` fields which are also marked as being
+:obj:`~typing.ReadOnly`, :obj:`~typing.Required`, or :obj:`~typing.NotRequired`
+(:issue:`4474`).
```

Comment:
it's actually a bit more than that: this fixes use of `Annotated` on `TypedDict`, period. This was broken before:

```
class ExampleDict(TypedDict):
    x: Annotated[int, Ge(0)]
```